changelogs/current.yaml

```diff
@@ -20,5 +25,9 @@ removed_config_or_runtime:
 # *Normally occurs at the end of the* :ref:`deprecation period <deprecated>`
 
 new_features:
+- area: upstream
+  change: |
+    Adds the option enable_full_scan on envoy.extensions.load_balancing_policies.least_request.v3.LeastRequest
+    so instead of using the choice_count to choose the hosts, we do a full scan on this list of hosts.
```

Comment:
```
Added :ref: `enable full scan option 
<envoy_v3_api_field_extensions.load_balancing_policies.least_request.v3.LeastRequest.enable_full_scan>`
to least request load balancer. If it is set to true, Envoy will do a full scan on the list of hosts rather than using
`choice_count
<envoy_v3_api_field_extensions. load_balancing_policies.least_request.v3.LeastRequest.choice_count>`.
```

I am not a native speaker, so you can also check the grammar by chatgpt or similiar API tools.

/wait